public/app/plugins/datasource/prometheus/components/monaco-query-field/monaco-completion-provider/util.ts

```diff
@@ -0,0 +1,9 @@
+// this class is used in a switch statement below,
```

Comment:
This is handy but since it was moved docs should be updated too, maybe with an example how to use it.

Reply:
good catch! i updated the comment with some example code.